### DISCUSSION TITLE
Version 2.3

### DISCUSSION THREAD

**Discussion by Jaysmito101** [Jaysmito101/TerraForge3D#36] [Announcements]
### Shader Node Editor
A blender like node editor for shaders.

### Texture Baker
A new and advanced way for baking and exporting Textures.

### Added Linux Support
Now TerraForge3D is supported natively on linux.
⚠️  Linux Version is in Alpha

### PBR Renderer
Now TerraForge3D has a Physically Based Renderer for high quality preview

### New Modules API
Now you can extend TerraForge3D however you want and write custom plugins using the powerful TerraForge3D SDK.

### Refactored Codebase
Not a big thing from users point of view but for anyone wanting to develop TerraForge3D its a huge deal. The codebase has been massively  cleaned up.

There has been several small fixes and additions which are not mentioned here.

## Some Relevant PRs
* Change directories in includes to lowercase by @lunaboards-dev in https://github.com/Jaysmito101/TerraForge3D/pull/26
* Fix pixel alignment kept at default when loading textures from STBI by @ltjax in https://github.com/Jaysmito101/TerraForge3D/pull/27
* Fix uniform IDs not being initialized by @ltjax in https://github.com/Jaysmito101/TerraForge3D/pull/28
* Added Doxygen Support by @Jaysmito101 in https://github.com/Jaysmito101/TerraForge3D/pull/29
* Linux build fixes. by @MalikuMane in https://github.com/Jaysmito101/TerraForge3D/pull/30
* Linux fix paths by @fiplox in https://github.com/Jaysmito101/TerraForge3D/pull/32

## New Contributors
* @lunaboards-dev made their first contribution in https://github.com/Jaysmito101/TerraForge3D/pull/26
* @ltjax made their first contribution in https://github.com/Jaysmito101/TerraForge3D/pull/27
* @MalikuMane made their first contribution in https://github.com/Jaysmito101/TerraForge3D/pull/30
* @fiplox made their first contribution in https://github.com/Jaysmito101/TerraForge3D/pull/32

**Full Changelog**: https://github.com/Jaysmito101/TerraForge3D/compare/v2.2...v2.3

<hr /><em>This discussion was created from the release <a href='https://github.com/Jaysmito101/TerraForge3D/releases/tag/v2.3'>Version 2.3</a>.</em>

**Comment by ghost** [comment]
This is my first time using the program and the initial linux release seems stable. No problems so far just need some time to figure out the interface. Thanks to the contributors.

**Reply by Jaysmito101** [reply]
Thanks!
